platform/saivpp/vpplib/SwitchStateBaseAcl.cpp

```diff
@@ -1274,11 +1613,21 @@ sai_status_t SwitchStateBase::aclBindUnbindPorts(
 	    continue;
 	}
 
-	if (is_bind)
+
+	if (is_bind) {
 	    ret = vpp_acl_interface_bind(hwif_name.c_str(), acl_swindex, is_input);
-	else
+        if(ret == 0) {
+            hwif_list.push_back(hwif_name);
+            tunterm_acl_bindunbind(tbl_oid, true, hwif_name);
+        }
+    }
+	else {
 	    ret = vpp_acl_interface_unbind(hwif_name.c_str(), acl_swindex, is_input);
-
+        if(ret == 0) {
+            hwif_list.remove(hwif_name);
```

Comment:
In the unbind case, wouldn't hwif_list always be empty?

Reply:
Updated